mod/metrics/set/abstract/metric_answer/flags.rb

```diff
@@ -25,6 +25,21 @@ def commented?
     end
   end
 
+  def calculated_flag
+    return "" unless card.calculated?
+    return hybrid_flag if card.hybrid?
```

Comment:
this is a metric method. Any researchable calculated metric is a hybrid.

You want `calculation_overridden?`

Let's not do this second `return`
Try:
```
if A
  B
else
  C
end
```